tests/requirements/django42.txt

```diff
@@ -0,0 +1 @@
+django>=4.2a1,<5.0
```

Comment:
I would change this to >=4.2,<5 for coherence with other versions. What do you think?

Reply:
Good point! Changed